distributions/aws/examples/storage/fsx-for-lustre/fsx-csi-driver.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
is this something we have crafted or copied ? If its copied please just point the,

Reply:
It is copied but unlike the efs one it is added as text to the github README file. I was planning to open an issue/create a PR to the fsx-csi repo to fix this so we could clone it as a file.

Reply:
Basically, pointing to either the github README or the official AWS docs is not a very clean redirect at the moment since they both have slightly confusing instructions about permissions. We need to make some changes upstream before we can do that. Take a look - 

1. [FSx Official Docs](https://docs.aws.amazon.com/eks/latest/userguide/fsx-csi.html)
2. [FSx Github Repo](https://github.com/kubernetes-sigs/aws-fsx-csi-driver#readme)

What we want is something like [EFS Github Repo](https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/release-1.3/docs/iam-policy-example.json) from where we can directly clone the file so user does not have to go through the upstream README.

Reply:
yes, can you please create an issue in their github repo, the discrepancy. 

either their doc or github repo one of them might be outdated.

Reply:
there is also discrapancy in information
https://github.com/kubernetes-sigs/aws-fsx-csi-driver#kubernetes-version-compability-matrix
vs
https://docs.aws.amazon.com/eks/latest/userguide/fsx-csi.html

```
This topic shows you how to deploy the FSx for Lustre CSI Driver to your Amazon EKS cluster and verify that it works. We recommend using version 0.4.0 of the driver.

Note
This driver is supported on Kubernetes version 1.21 and later Amazon EKS clusters and nodes.
```